packages/kbn-plugin-helpers/package.json

```diff
@@ -20,5 +20,8 @@
     "through2": "^2.0.3",
     "through2-map": "^3.0.0",
     "vinyl-fs": "^3.0.0"
+  },
+  "devDependencies": {
+    "node-sass": "^4.9.0"
```

Comment:
kbn-plugin-helpers is a devDependency, so things that would normally be devDependencies in Kibana should be regular dependencies here.